src/orders/components/OrderTransaction/utils.ts

```diff
@@ -16,3 +23,121 @@ export const mapActionToMessage: Record<
   CANCEL: transactionActionMessages.cancel,
   REFUND: null,
 };
+
+export const mapTransactionEvent = (
+  event: TransactionEventFragment | TransactionFakeEvent,
```

Comment:
suggestion: event should be undefined | TransactionEventFragment | TransactionFakeEvent